CONTRIBUTING.md

```diff
@@ -24,7 +24,9 @@ For more information, please see STYLEGUIDE.md
 
 # <a name="Attachments"></a>Attachments
 
-GitHub only allows image attachments. Other attachments, for example zipped logs, can be posted in the <a href="http://www.universalmediaserver.com/forum/viewtopic.php?f=14&t=1656">Github attachments thread</a> in the UMS forum.
+GitHub only allows few attachments formats. Other attachments, for example zipped logs, can be posted in the <a href="http://www.universalmediaserver.com/forum/viewtopic.php?f=14&t=1656">Github attachments thread</a> in the UMS forum.
```

Comment:
As everyone can see it by themself i didn't intended to write them all, but leave only the ZIP example.
I hope that you're sure about "a few" as for me that look strange ;)